proto/umee/oracle/v1/query.proto

```diff
@@ -252,6 +252,8 @@ message QueryMedians {
   option (gogoproto.goproto_getters) = false;
   // denom defines the denomination to query for.
   string denom = 1;
+  // numStamps defines the number of median stamps to query for.
+  uint64 numStamps = 2;
```

Comment:
is there any lower and upper bound for this param

Reply:
I added a check in the Medians query method that the numStamps parameter is not greater than the MaximumMedianStamps param so that the bound is based to what that param is set to.